### DISCUSSION TITLE
Issue when migrating DB changes in production on AWS

### DISCUSSION THREAD

**Discussion by lequydinh** [prisma/prisma#13456] [Q&A] [answered]
Hi everyone.
I'm deploying an app with CodeBuild, RDS, and ECS. My CICD (CodeBuild) is in one AWS account, and other services (RDS and ECS) are in another AWS account. I'm not sure where I should put the **npx prisma migrate deploy** in the CICD to migrate the changes to the DB in RDS? Is it possible to only run it after the ECS container deploys successfully?
Thank you.

**Comment by nurul3101** [comment, marked as answer]
Hey @lequydinh 👋 
It's recommended that the `npx prisma migrate deploy` command should be a part of an automated CI/CD pipeline, and we do not generally recommend [running this command locally to deploy changes to a production database](https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-migrations-from-a-local-environment) (for example, by temporarily changing the DATABASE_URL environment variable). It is not generally considered good practice to store the production database URL locally.

Here's the reference on how database changes should be deployed with Prisma Migrate: [Reference](https://www.prisma.io/docs/guides/deployment/deploy-database-changes-with-prisma-migrate)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
